.storybook/preview.js

```diff
@@ -12,7 +13,7 @@ export const parameters = {
 
 export const decorators = [
   (Story) => (
-    <ThemeProvider>
+    <ThemeProvider theme={defaultTheme}>
```

Comment:
We should only pass the `theme` once so then it's provided to all components, rather than each component getting the theme as a prop. Refactored all the components with this PR, let me know your thoughts please.

Reply:
First thought: this is way better
Second thought: what if they want one specific theme ?
Third thought: Naaa, this is way better